api/views/quiz_views.py

```diff
@@ -96,7 +96,7 @@ class InstructorQuizzesView(APIView):
 
     @extend_schema(responses={200: QuizListSerializer}, summary="Get all quizzes by instructor")
     def get(self, request):
-        quizzes = Quiz.objects.filter(instructor=request.user.instructor)
+        quizzes = Quiz.objects.filter(instructor=request.user.instructor).order_by("-created_at")
```

Comment:
It would be better/cleaner if we define the ordering in `models.py` (in `class Meta`)